webpack.config.demo.js

```diff
@@ -1,38 +1,41 @@
 const path = require('path');
-const HtmlWebpackPlugin = require("html-webpack-plugin");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 const htmlWebpackPlugin = new HtmlWebpackPlugin({
-    template: path.join(__dirname, "./example/src/index.html"),
-    filename: "./index.html"
+  template: path.join(__dirname, './example/src/index.html'),
+  filename: './index.html'
 });
 
 module.exports = {
-
-  entry: path.join(__dirname, "./example/src/app.jsx"),
+  entry: path.join(__dirname, './example/src/app.jsx'),
   output: {
-      path: path.join(__dirname, "example/dist"),
-      filename: "bundle.js"
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js'
   },
   module: {
     rules: [
-        // All files with a '.ts' or '.tsx' extension will be handled by 'awesome-typescript-loader'.
-        { test: /\.tsx?$/, loader: "ts-loader" },
-        {
-          test: /\.(js|jsx)$/,
-          use: "babel-loader",
-          exclude: /node_modules/
-        },
-        {
-          test: /\.css$/,
-          use: ['style-loader', 'css-loader'],
-        }
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader'
+      },
+      {
+        test: /\.(js|jsx)$/,
+        use: 'babel-loader',
+        exclude: /node_modules/
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader'
+        ],
+      }
     ]
-},
+  },
   plugins: [htmlWebpackPlugin],
   resolve: {
-    // Add '.ts' and '.tsx' as resolvable extensions.
-    extensions: [".ts", ".tsx", ".js", ".json"]
-},
+    extensions: ['.ts', '.tsx', '.js', '.json']
```

Comment:
Prefer double quotes in files for strings please 😄

Reply:
I can also update this setting in #79 in the linter and formatter configs. Would you prefer double quotes in the whole project as it's not consistent at the moment?

Reply:
Done

Reply:
Yeah, ideally through the whole project but happy to merge this and accept the double quote change in #79 as it seems more appropriate there anyway.